lisp/ess-inf.el

```diff
@@ -59,6 +59,13 @@
 (declare-function ess-mode "ess")
 (declare-function ess-complete-object-name "ess-r-completion")
 
+;; The following declares can be removed once we drop Emacs 25
+(declare-function tramp-file-name-method "tramp")
+(declare-function tramp-file-name-user "tramp")
```

Comment:
Cannot we just require tramp?

Reply:
No, it takes too long to load if we don't need it (see the commit message in 30a3acf7e18f87ee7541275d2e7198769f1c633e)